use_cases.md

```diff
@@ -81,7 +83,8 @@ and [others]({{ site.baseurl }}/powered_by/) also use Arrow similarly.
 
 The Arrow project also defines [Flight]({% post_url 2019-09-30-introducing-arrow-flight %}),
 a client-server RPC framework to build rich services exchanging data according
-to application-defined semantics.
+to application-defined semantics. Flight RPC is used by [UKV](https://unum.cloud/ukv)
+to exchange tables, documents, and graphs, between server application and client SDKs.
```

Comment:
Could you revert this? We refer the `powered_by/` page in the above paragraph. UKV is introduced in the page.

Reply:
You mean removing the link duplicate, or the whole contents of the line?

Reply:
I mean that reverting all changes in `use_cases.md`.

Reply:
Reverted all the changes in `use_cases.md`.

Apache Spark, Google BigQuery, TensorFlow, and AWS Athena were all mentioned in that paragraph, so I thought it might be the right place to mention them. We rely on Arrow representations for the same purpose but with a much broader scope than any of mentioned projects. Maybe we can add the reference another time. Thank you!

Reply:
Apache Arrow project provides https://arrow.apache.org/powered_by/ as a page for introducing third-party projects including their use case. It's mentioned explicitly:

> To add yourself to the list, please open a [pull request](https://github.com/apache/arrow-site/edit/master/powered_by.md) adding your organization name, URL, a list of which Arrow components you are using, and a short description of your use case.

But other pages such as  https://arrow.apache.org/use_cases/ haven't discussed for the purpose explicitly. If you think that Apache Arrow project should use https://arrow.apache.org/use_cases/ for the purpose too, could you start a discussion on `dev@arrow.apache.org` mailing list?

https://arrow.apache.org/community/

> dev@ is for discussions about contributing to the project development ([subscribe](mailto:dev-subscribe@arrow.apache.org?subject=Subscribe), [unsubscribe](mailto:dev-unsubscribe@arrow.apache.org?subject=Unubscribe), [archives](https://lists.apache.org/list.html?dev@arrow.apache.org))

FYI: Apache Software Foundation provides suggested practices related to this topic: https://www.apache.org/foundation/marks/linking